scripts/dcp1-version/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/ebi-ait/ingest-base-images:python_3.7-slim
+
+COPY ./populate_dcp_version.py populate_dcp_version.py
```

Comment:
copying the code should be after copying the requirements file and installing requirements. This speeds up the docker build because it would use the docker cache more efficiently.

Reply:
Yup these changes were moved to ingest-kube-deployment repo 
PR: https://github.com/ebi-ait/ingest-kube-deployment/pull/36 

new dockerfile: https://github.com/ebi-ait/ingest-kube-deployment/blob/master/jobs/dcp1-version/docker-app/Dockerfile

And thanks! I am reminded that I should delete this directory in this repo.